domains/sahil.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal Porfolio Records Add.",
+  "repo": "https://github.com/sahilpatel09/sahilpatel09.github.io",
+  "owner": {
+    "username": "sahil",
```

Comment:
gh username pls?

Reply:
I have added the github username. Sorry forgot it.